katsdpcontroller/generator.py

```diff
@@ -344,15 +343,15 @@ def _make_fgpu(
             '--src-affinity', '{cores[src0]},{cores[src1]}',
             '--dst-interface', '{interfaces[cbf].name}',
             '--dst-affinity', '{cores[dst]}',
-            # TODO: reenable once katxgpu can handle it
+            # TODO: reenable once katgpucbf.xbgpu can handle it
             # '--dst-packet-payload', '8192',
```

Comment:
Is there a JIRA ticket or some TODO note somewhere to remind us to do this?

Reply:
NGC-208.